build.sbt

```diff
@@ -636,31 +684,23 @@ lazy val publishSettings = Seq(
   )
 ) ++ credentialSettings ++ sharedPublishSettings ++ sharedReleaseProcess
 
-// These aliases serialise the build for the benefit of Travis-CI.
+// Scalafmt
+addCommandAlias("fmt", "; compile:scalafmt; test:scalafmt; scalafmtSbt")
+addCommandAlias("fmtCheck", "; compile:scalafmtCheck; test:scalafmtCheck; scalafmtSbtCheck")
 
+// These aliases serialise the build for the benefit of Travis-CI.
 addCommandAlias("buildKernelJVM", ";kernelJVM/test;kernelLawsJVM/test")
-
 addCommandAlias("buildCoreJVM", ";macrosJVM/test;coreJVM/test")
-
 addCommandAlias("buildTestsJVM", ";lawsJVM/test;testkitJVM/test;testsJVM/test;jvm/test")
-
 addCommandAlias("buildFreeJVM", ";freeJVM/test")
-
 addCommandAlias("buildAlleycatsJVM", ";alleycatsCoreJVM/test;alleycatsLawsJVM/test;alleycatsTestsJVM/test")
-
 addCommandAlias("buildJVM", ";buildKernelJVM;buildCoreJVM;buildTestsJVM;buildFreeJVM;buildAlleycatsJVM")
-
 addCommandAlias("validateBC", ";binCompatTest/test;mimaReportBinaryIssues")
-
-addCommandAlias("validateJVM", ";scalastyle;buildJVM;bench/test;validateBC;makeMicrosite")
-
+addCommandAlias("validateJVM", ";buildJVM;bench/test;validateBC;makeMicrosite")
 addCommandAlias("validateJS", ";catsJS/compile;testsJS/test;js/test")
-
 addCommandAlias("validateKernelJS", "kernelLawsJS/test")
-
 addCommandAlias("validateFreeJS", "freeJS/test") //separated due to memory constraint on travis
-
-addCommandAlias("validate", ";clean;validateJS;validateKernelJS;validateFreeJS;validateJVM")
+addCommandAlias("validate", ";clean;scalastyle;fmtCheck;validateJS;validateKernelJS;validateFreeJS;validateJVM")
```

Comment:
I think that it might be better to move the scalastyle and format check back to `validateJVM`. While it's not JVM-specific, our build runs `validateJVM` or `validateJS` depending on whether it's a JVM or JS build (combining both in a single build leads to more memory issues). So to make sure that this is run, I think that we need to choose one for it to run in, and `validateJVM` is what we were using.

Our builds probably need a bit of an overhaul, but I don't think that we should try to tackle that in a PR that has this big of a changeset, or we'll never get either merged :)

Reply:
👍

Reply:
I reverted this